.circleci/config.yml

```diff
@@ -47,8 +48,42 @@ commands:
             # "!" negates return code. exit nonzero if any of these deps are found
             ! pip freeze | grep -E "xgboost|catboost|lightgbm|plotly|ipywidgets"
             exit $?
+  build_pkg:
+    steps:
+      - run: |
+            git clone https://github.com/conda-forge/evalml-core-feedstock
+            source test_python/bin/activate
+            mkdir evalml-core-feedstock/evalml
+            cp -r `ls -A | grep -v "evalml-core-feedstock"` ./evalml-core-feedstock/evalml/
+            python conda_config.py "$(python setup.py --version)"
+            cd evalml-core-feedstock
+            echo "$DOCKER_HUB_PASS" | docker login -u psalter --password-stdin
+            export DOCKER_CONTAINERID="$(docker run -td condaforge/linux-anvil-comp7)"
+            echo "Created container ${DOCKER_CONTAINERID}"
+            chmod -R 777 ./
+            docker cp . ${DOCKER_CONTAINERID}:/home/conda/feedstock_root/
+            docker cp ./recipe/. ${DOCKER_CONTAINERID}:/home/conda/recipe_root/
+            echo "COMMITING UPDATED IMAGE"
+            docker commit ${DOCKER_CONTAINERID} psalter/build:latest
+            docker stop ${DOCKER_CONTAINERID}
+            export CONFIG=linux_64_
+            export UPLOAD_PACKAGES=False
+            export HOST_USER_ID=$(id -u)
+            export FEEDSTOCK_NAME=evalml-core-feedstock
+            docker run -t -e CONFIG -e HOST_USER_ID -e UPLOAD_PACKAGES -e GIT_BRANCH -e UPLOAD_ON_BRANCH -e CI -e FEEDSTOCK_NAME -e CPU_COUNT -e BINSTAR_TOKEN -e FEEDSTOCK_TOKEN -e STAGING_BINSTAR_TOKEN psalter/build:latest bash /home/conda/feedstock_root/.scripts/build_steps.sh
 
 jobs:
+  build_conda_pkg:
+    working_directory: ~/evalml/
+    executor:
+      name: python
+      python_version: "3.6"
```

Comment:
@ryan-s @freddyaboulton why python 3.6? Should we do this for all our supported python versions, 3.6, 3.7 and 3.8? Or just the latest version, 3.8?

Reply:
I think this sets the version of python in the circleci worker. The docker run command will actually build the package in python 3.7.8 - this is the same as what happens in the CI of our feedstock repo. Unfortunately, there isn't an out-of-the-box way to build with different python versions but I will add this to the future work section of our design doc for this issue.

Reply:
Ah got it. So we'll only build and run our tests on 3.7.8 for now. No problem, this is a great starting point. Thanks!

Reply:
Correct, I just needed something to install pyyaml into so that we can update the recipe.  

I think the v2 of this would be to create our own container(s) based off of condaforge/linux-anvil-comp7 where we can move the existing functionality and any new improvements into.